docs/intro.asciidoc

```diff
@@ -6,17 +6,23 @@ endif::[]
 [[intro]]
 == Introduction
 
-Welcome to the APM .NET Agent documentation.
+The Elastic APM .NET Agent automatically measures the performance of your application and tracks errors.
+It has built-in support for the most popular frameworks and routers,
```

Comment:
What is `routers`? I'm not sure it has a well known meaning in .NET. 
Update: Hah, by looking at other docs, I suspect that's some Node.js lingo :) Probably I'd still not add that word here.

Reply:
Ahhh, oops! Sorry about that